[Docs] Switch steps 3 and 4 in M1 installation instructions (#35513)
ray-project/ray: doc/source/ray-overview/installation.rst

   * ``conda activate``

#. Install Ray as you normally would.

   * ``pip install ray``

> #. Ensure that the ``grpcio`` package is installed via forge and **not pypi**. Grpcio currently requires special compilation flags, which pypi will _not_ correctly build with. Miniforge provides a prebuilt version of grpcio for M1 macs.

Reviewer: ```suggestion
#. Ensure that the ``grpcio`` package is installed via forge and **not pypi**. Grpcio requires special compilation flags, which pypi does _not_ correctly build with. Miniforge provides a prebuilt version of grpcio for M1 macs.
```
